### PR TITLE
Remove debug logging + minor cleanups

### PR DESCRIPTION
**Pull request by timja**
### Testing done

Loaded my user with a gravatar and it works fine

### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/gravatar/boundary/GravatarImageURLVerifier.java [hunk 1]
```diff
@@ -90,8 +88,7 @@ protected Gravatar gravatar() {
     }
 
     private boolean responseCodeIsOK(int gravatarResponseCode) {
-        System.err.println("gravatarResponseCode: " + gravatarResponseCode);
```
Comment (timja) [inline src/main/java/org/jenkinsci/plugins/gravatar/boundary/GravatarImageURLVerifier.java:93]: Looks accidentally left in when http client was changed

File: src/main/java/org/jenkinsci/plugins/gravatar/factory/GravatarFactory.java [hunk 2]
```diff
@@ -19,11 +19,6 @@ public Gravatar testGravatar() {
     }
 
     private Gravatar gravatar() {
-        return new Gravatar().setHttps(useHttps());
```
Comment (timja) [inline src/main/java/org/jenkinsci/plugins/gravatar/factory/GravatarFactory.java:22]: noticed when I develop locally I was getting an http url which didn't seem right, I can't see why you would not want to fetch this over https

Comment (viceice) [reply src/main/java/org/jenkinsci/plugins/gravatar/factory/GravatarFactory.java:22]: I didn't change it for tests yet. you'll now need to fix the failing test to get this merged

**Review by viceice — COMMENTED** [review]
LGTM besides missing test fix

**Review by viceice — APPROVED** [review]

**Pull request merged**